scripts/status.sh

```
@@ -0,0 +1,145 @@
+YELLOW='\033[1;33m'
+CYAN='\033[0;36m'
+NC='\033[0m'
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+
+print_running() { echo -e "  ${GREEN}● RUNNING${NC}   $1"; }
+print_stopped() { echo -e "  ${RED}● STOPPED${NC}   $1"; }
+print_header() { echo -e "${CYAN}$1${NC}"; }
+
+cd "$PROJECT_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)